src-cpp/rdkafkacpp.h

```diff
@@ -1049,6 +1049,23 @@ class RD_EXPORT Handle {
    * @returns ERR_NO_ERROR on success or an error code on error.
    */
   virtual ErrorCode set_log_queue (Queue *queue) = 0;
+
+  /**
+   * @brief Returns the ClusterId as reported in broker metadata.
+   *
+   * @param timeout_ms If there is no cached value from metadata retrieval
```

Comment:
Fixed some grammar, but this piece of documentation belongs here.
Can you clarify the confusion?